src/utils/hooks/useRouter.tsx

```diff
@@ -34,6 +35,17 @@ interface RouterResult {
   location: any
 }
 
+const replaceHifenKeys = (parsedQuery: { [key: string]: any }) => {
```

Comment:
```suggestion
const replaceSnakeCaseKeys = (parsedQuery: { [key: string]: any }) => {
```
I assume you meant "hyphen", right? I'd prefer `replaceSnakeCaseKeys`, but if not it'd be `replaceHyphenKeys` ;)

Reply:
Also, this is the kind of function that would really suit some unit tests. I don't think we have Jest installed on front-end yet, but maybe make an issue to configure Jest and add this as one of the functions we should write tests for.

Reply:
Actually, I put you wrong: apparently hyphens is `kebab-case`. `snake_case` is underscores. 🤦‍♂️ 😆